### DISCUSSION TITLE
Can only connect to Rustdesk's servers with VPN

### DISCUSSION THREAD

**Discussion by Shadow-Phax** [rustdesk/rustdesk#5863] [General]
For the last two days I've not been able to connect to the Rustdesk severs except when I connect through a VPN. I never had this issue before. It appears my IP is being blocked for some reason.

**Comment by Piscean** [comment]
I also lost connectivity to Rustdesk servers last night (PST).  

Looks like the server is down?
c:\>ping rs-ny.rustdesk.com

Pinging rs-ny.rustdesk.com [155.138.247.159] with 32 bytes of data:
Request timed out.
Request timed out.
Request timed out.
Request timed out.

**Reply by Shadow-Phax** [reply]
I'm pretty sure they're blocking my IP for some reason because I have two machines connected to the Rustdesk servers from two different VPN locations. They're talking to each other and working normally.

**Reply by Gangstrocity** [reply]
I have rustdesk on a linux VM that runs through a VPN. It can't connect anymore.

**Comment by EZhelp** [comment]
Confirmed same issue as Shadow-Phax.   My normal WAN IP (from Xfinity) seems to be blocked from the Rustdesk servers.  If I ping rs-ny.rustdesk.com ,  request timed out / no response.   If I connect to a vpn and then ping the same address, I get 4 fast 65 ms responses.  I re-tested using multiple VPNs on multiple computers. Same results.   I know other Rustdesk users having issues (with Xfinity ISP))  over the past few days and had them test with and without using a vpn with the same results as I had.   I have also tested from my normal WAN IP using several DNS servers and then the NY server ip address directly 155.138.247.159 with the same time out results.  So, its not a dns issue. The free public server has been unreachable from here for about 4 days now.

Edit... 10/2/2023... more testing.   See post below

In the meantime, I have configured a private self hosted server which is working well.

**Reply by Shadow-Phax** [reply]
Thank you for your responses. I see now that it's a pretty widespread problem and I haven't seen and I hope it gets fixed soon or they let us know what's going on. I was thinking of setting up a private host server and might in the future. I have been using TeamViewer but they put new limits in and I'll be dumping them. I'm currently using UltraVNC, Chrome Remote Desktop and DWservice. DWService is the most promising. Very easy to install and works with Windows and Linux among others.

**Comment by EZhelp** [comment]
More info and testing.    I went to a neighbors house with Xfinity internet (they have never used Rustdesk and it has never been installed)  I used their Xfinity internet connection to ping the RS-NY server at 155.138.247.159... result timed out... 100% packet loss.  If I run tracert 155.138.247.159 it makes it to a 4th hop at 96.216.61.57 then goes dead.  A search indicates 96.216.61.57 is a comcast device in Seattle WA.   If I connect to a vpn (not xfinity) I can ping with success and rustdesk works.  

So now my best guess is Xfinity/Comcast and maybe other ISPs are not able to reach the host of Rustdesk server in NY?  This is not an issue with the Rustdesk software.  Its either a Xfinity/Comcast issue... or... whoever is hosting the public server in NY has issues??
It would be nice to resolve the issue as many new users will try the public server as a trial... with bad results.

**Comment by Shadow-Phax** [comment]
I just tested Rustdesk on computer connected to the Internet via my cellphone and it worked just fine. Rustdesk quickly connected to the Rustdesk sever. I'm starting to think that maybe Xfinity/Comcast is blocking Rustdesk server connections because it's been used in the past in scams to get personal info. These scammers get the unsuspecting person to install Rustdesk and then use it to gather personal information. I'm going to look into this more to see what I can find out.

**Comment by Gangstrocity** [comment]
Rust desk works fine on my computer. I'm using mullvad VPN so I can quickly select servers all over the world. It works on most of them, but from some reason the primary server that I have a linux computer connected to is now blocked. I can send my regular computer through that same VPN and watch rustdesk lose connection.

Anyone have good documentation for setting up a relay server? I managed to get it working, but it only works on my network. I need to be able to access these computers when I'm away. I have my own domain and a reverse proxy set up which I think may be causing some issues.
